local.properties

```diff
@@ -0,0 +1,11 @@
+## This file is automatically generated by Android Studio.
+# Do not modify this file -- YOUR CHANGES WILL BE ERASED!
+#
+# This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Tue Sep 13 09:24:40 CEST 2016
+sdk.dir=/Users/colin/Library/Android/sdk
```

Comment:
@csdodd Many thanks for this PR! 
Just one thing: can you please remove this file?